test/opt/test_pass_manager.cpp

```diff
@@ -66,6 +79,24 @@ class AppendOpNopPass : public opt::Pass {
   }
 };
 
+// A pass that appends specified number of OpNop instructions to the debug
+// section.
+class AppendMultipleOpNopPass : public opt::Pass {
+ public:
+  AppendMultipleOpNopPass(uint32_t num_nop) : num_nop_(num_nop) {}
+  const char* name() const override { return "AppendOpNop"; }
+  bool Process(ir::Module* module) override {
+    for (uint32_t i = 0; i < num_nop_; i++) {
+      auto inst = MakeUnique<ir::Instruction>();
+      module->AddDebugInst(std::move(inst));
```

Comment:
Hmm, I think that would make the line too long. I think the current form is easier to read